docs/marks.rst

```diff
@@ -113,6 +113,33 @@ You can then run tests that are relevant only to the selected ticket:
     cd src/tests/system
     pytest --mh-config=mhc.yaml --mh-lazy-ssh -v --ticket=gh#3433
 
+@pytest.mark.importance
+=======================
```

Comment:
I'm missing a guide how to clasify the tests. What makes the test "critical", what makes it "low"?

Reply:
How is QA using this marker? It doesn't really make sense form upstream point of view, so this should be also clarified.